static/css/static.css

```diff
@@ -1748,4 +1748,4 @@ body.interface-hebrew #visualGardenPage .filter-title .reset {
 }
 .phone-number{
   unicode-bidi: plaintext;
-}
\ No newline at end of file
+}
```

Comment:
Most unix editors demand this change, it's possible to suppress, but it'd be nicer if the entire corpus had it so that such changes don't randomly appear in PRs